docs/source/optimizations/cq/cq_compile_css.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "eb89a144",
```

Comment:
<p>Doesn't have to be very verbose but would be nice to have similar description and commentary for each of the codeblocks in this notebook so the user can have a better understanding of what's going on.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done</p>